robocop/checkers/lengths.py

```diff
@@ -197,9 +203,12 @@ def __init__(self):
         self.max_line_length = 120
         # replace # noqa or # robocop, # robocop: enable, # robocop: disable=optional,rule,names
         self.disabler_pattern = re.compile(r'(# )+(noqa|robocop: ?(?P<disabler>disable|enable)=?(?P<rules>[\w\-,]*))')
+        self.ignore_pattern = re.compile(r'https?://\S+')
```

Comment:
Shouldn't those backslashes be escaped like `r'https?:\/\/\S+'`?

Reply:
No, thanks to r'